<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartIntercom - вход</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>448</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>448</height>
    </size>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="main_label">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>76</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>76</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>8</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Панель администратора&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_t">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>57</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>57</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Логин:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>39</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>39</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="passw_t">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>56</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Пароль:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passw">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>39</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>39</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="error_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auth_button">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>601</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
